reana_commons/openapi_specifications/reana_server.json

```diff
@@ -574,6 +679,18 @@
                   "type": "object"
                 }
               },
+              "required": [
+                "compute_backends",
+                "default_kubernetes_jobs_timeout",
+                "default_kubernetes_memory_limit",
+                "default_workspace",
+                "kubernetes_max_memory_limit",
+                "maximum_interactive_session_inactivity_period",
+                "maximum_kubernetes_jobs_timeout",
+                "maximum_workspace_retention_period",
+                "workspaces_available",
+                "dask_enabled"
```

Comment:
Note that testing the `reana-client` with this `reana-commons` PR does not work against "older" REANA clusters that do not contain Dask support, I mean even with non-Dask workflows, for example:

```console
$ cd reana-demo-root6-roofit
$ export REANA_SERVER_URL=https://reana.cern.ch
$ read -s REANA_ACCESS_TOKEN
$ reana-client create -w test
==> ERROR: Cannot create workflow test:
'dask_enabled' is a required property
```

It would be good to make the client workable against both "old" (Dask-unaware) and "new" (Dask-aware) clusters.

Basically, we have four cases in the test matrix to consider: old client <-> old cluster, old client <-> new cluster, new client <-> old cluster, new client <-> new cluster.  The Dask support is "optional" and so ideally all these combinations should keep working.

Reply:
I removed the `required` part from the openapi spec and now it should be working fine with all combinations. I tested with several examples and all should be working fine.